### PR TITLE
Update to 1.9.7

### PR DESCRIPTION
**Pull request by MaDDoGo**
### Description
- Fix an issue with some dependency plugins were causing a lock of Kodi
- Improved search a little bit more
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scrapers/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: metadata.filmaffinity.com/.github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (razzeee) [inline metadata.filmaffinity.com/.github/workflows/main.yml:1]: Can you drop this file?

**Comment by razzeee** [comment]
You can use this script to convert your language files https://gitlab.com/ronie/script.language-convert

**Comment by MaDDoGo** [comment]
Hi,

I tried to execute your script, but it fails from some encoding reason (don't know if it's on your files or mine, I downloaded both from repository as zip and installed right away

```
text = parse.unescape(string.text)
                                                   
ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
 - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <class 'UnicodeEncodeError'>
Error Contents: 'ascii' codec can't encode character '\xf3' in position 1170: ordinal not in range(128)
Traceback (most recent call last):
  File "/Users/me/Library/Application Support/Kodi/addons/script.language-convert/default.py", line 162, in <module>
    Main()
  File "/Users/me/Library/Application Support/Kodi/addons/script.language-convert/default.py", line 30, in __init__
    self.updateFiles(langpath, folders, name, addonid, author)
  File "/Users/me/Library/Application Support/Kodi/addons/script.language-convert/default.py", line 129, in updateFiles
    f.write(newText)
UnicodeEncodeError: 'ascii' codec can't encode character '\xf3' in position 1170: ordinal not in range(128)
-->End of Python script error report<--
```

**Comment by razzeee** [comment]
You'll need to report that at the repo mentioned, I can't offer support for that here.

**Comment by enen92** [comment]
Ping @MaDDoGo

**Comment by MaDDoGo** [comment]
I will close this one and reopen!

**Pull request closed without merging**